website/content/en/docs/cli/operator-sdk_bundle_validate.md

```diff
@@ -31,19 +31,19 @@ The following command flow will generate test-operator bundle image manifests
 and validate them, assuming a bundle for 'test-operator' version v0.1.0 exists at
 <project-root>/deploy/olm-catalog/test-operator/manifests:
 
-  # Generate manifests locally.
-  $ operator-sdk bundle create --generate-only
+  # Generate manifests and metadata locally.
+  $ operator-sdk generate bundle
 
   # Validate the directory containing manifests and metadata.
   $ operator-sdk bundle validate ./deploy/olm-catalog/test-operator
```

Comment:
Have we not more the options to choose the container-tool to validate? [podman, end etc]

Reply:
Yes, and they're listed in this doc below.